packages/relic/test/headers/new/special/accept_encoding_header_test.dart

```diff
@@ -0,0 +1,331 @@
+import 'dart:io';
+import 'package:test/test.dart';
+import 'package:relic/src/headers/headers.dart';
+import 'package:relic/src/relic_server.dart';
+import '../../headers_test_utils.dart';
+
+void main() {
+  group('Given an Accept-Encoding header with the strict flag true', () {
+    late RelicServer server;
+
+    setUp(() async {
+      try {
+        server = await RelicServer.bind(InternetAddress.loopbackIPv6, 0);
+      } on SocketException catch (_) {
+        server = await RelicServer.bind(InternetAddress.loopbackIPv4, 0);
+      }
+    });
+
+    tearDown(() => server.close());
+
+    test(
+      'when a valid Accept-Encoding header is passed then it should parse correctly',
+      () async {
+        Headers headers = await getServerRequestHeaders(
+          server: server,
+          headers: {'accept-encoding': 'gzip, deflate, br'},
+          strictHeaders: true,
+        );
+
+        expect(
+          headers.acceptEncoding?.encodings.map((e) => e.encoding).toList(),
+          equals(['gzip', 'deflate', 'br']),
+        );
+        expect(
+          headers.acceptEncoding?.encodings.map((e) => e.quality).toList(),
+          equals([1.0, 1.0, 1.0]),
+        );
```

Comment:
Suggestion: Separate the default value validation from the "encodings is parsed correctly" test.